test/upgrade/helpers.bash

```diff
@@ -1,6 +1,7 @@
 # -*- bash -*-
 
 load "../system/helpers"
+load "../system/helpers"	# For random_free_port()
```

Comment:
same line twice?

Reply:
Oops, right, that's why the upgrade test in the CI would still fail. Fixed.